engineio/benches/engineio.rs

```diff
@@ -45,6 +40,9 @@ pub mod util {
     }
 }
 
+/*            sync benches            */
+
+#[cfg(not(feature = "async"))]
```

Comment:
suggestion: why are tests turned off but the structs remain on when async is enabled?

Reply:
As far as I know the structs are not included, the mod.rs file contains `cfg(feature ...)` attributes.